Big-Spender/readme.md

```diff
@@ -112,7 +127,18 @@ INSERT YOUR QUERY HERE
 **You:** Then you need an extra clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+CREATE YOUR TABLE HERE
+
+SELECT '2021-03-01' AS date,
+       SUM(amount) AS total_amount
+FROM spends
+WHERE date = '2021-03-01';
+
+SELECT '2021-04-01' AS date,
+       SUM(amount) AS total_amount
+FROM spends
+WHERE date = '2021-04-01';
+
```

Comment:
so this would be two different queries? What if I want this to be one table? I'd want the columns: total, and date with two rows( the two dates)